### PR TITLE
Companion for substrate#9115

### PR DESCRIPTION
**Pull request by koushiro**
Companion for paritytech/substrate#9115

migrate the `Council` and `TechnicalCommittee` storage prefix:

`Instance1Collective` ==> `Council`
`Instance2Collective` ==> `TechnicalCommittee`

### CODE REVIEW

**Review by thiolliere — APPROVED** [review]
looks good to me, I tested the migration with try-runtime on polkadot block 6,163,183

**Comment by koushiro** [comment]
> looks good to me, I tested the migration with try-runtime on polkadot block 6,163,183

I want to learn about the `try-runtime` tool. Is there a corresponding tutorial or document?

File: runtime/kusama/src/lib.rs [hunk 1]
```diff
@@ -1525,7 +1525,12 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPallets,
-	(RemoveCollectiveFlip, MigratePalletVersionToStorageVersion),
+	(
+		RemoveCollectiveFlip,
+		MigratePalletVersionToStorageVersion,
+		CouncilStoragePrefixMigration,
+		TechnicalCommitteeStoragePrefixMigration
```
Comment (thiolliere) [inline runtime/kusama/src/lib.rs:1532]: I mixed up the merge, those migration are incompatible, the latter migration are checking pallet version, while the former is writing the new version into storage.

**Review by thiolliere — CHANGES_REQUESTED** [review]

**Comment by thiolliere** [comment]
> > looks good to me, I tested the migration with try-runtime on polkadot block 6,163,183
> 
> I want to learn about the `try-runtime` tool. Is there a corresponding tutorial or document?

there is this seminar https://www.crowdcast.io/e/substrate-seminar/41
and this doc https://substrate.dev/docs/en/knowledgebase/advanced/try-runtime
most important is to compile with try-runtime and then use the subcommand associated

**Review by thiolliere — APPROVED** [review]

**Comment by kianenigma** [comment]
/try-runtime queue --execution Native -lremote-ext=debug,executor=trace,runtime=trace --block-at 0xa7930a1fb28a1049ddfd50ea953674f9adf2d715b33a0b0a7678154e1bfba312 --url ws://polkadot on-runtime-upgrade live

**Review by kianenigma — APPROVED** [review]
Given that migration is tested.

**Comment by thiolliere** [comment]
I run a successful migration with try-runtime with on current kusama and polkadot

**Review by shawntabrizi — APPROVED** [review]

**Comment by thiolliere** [comment]
bot merge

**Comment by thiolliere** [comment]
bot merge

**Pull request merged**